kernel_gateway/services/kernels/manager.py

```diff
@@ -3,12 +3,15 @@
 """Kernel manager that optionally seeds kernel memory."""
 
 from notebook.services.kernels.kernelmanager import MappingKernelManager
+from jupyter_client.ioloop import IOLoopKernelManager
 from ..cell.parser import APICellParser
 
 class SeedingMappingKernelManager(MappingKernelManager):
     """Extends the notebook kernel manager to optionally execute the contents
     of a notebook on a kernel when it starts.
     """
+    kernel_manager_class = 'kernel_gateway.services.kernels.manager.KernelGatewayIOLoopKernelManager'
```

Comment:
Need to verify that this isn't accidentally creating a named class variable shadow.